packages/core/src/utils/path.ts

```diff
@@ -0,0 +1,20 @@
+import { platform } from "node:os";
+import { win32 } from "node:path";
+
+/**
+ * The function appends `file:///` protocol to the absolute paths on Windows due to ES modules imports specifics
+ * @param path
+ */
+export const normalizeImportPath = (path: string) => {
```

Comment:
Better to normalize paths with `url.pathToFileURL` as [Node.js docs suggests](https://nodejs.org/api/esm.html#file-urls). We should use it even on mac/linux because node requires module specifiers to be percent-encoded, which `pathToFileURL` does.